sys/fs/fuse/fuse_vnops.c

```diff
@@ -3122,10 +3128,10 @@ fuse_vnop_deleteextattr(struct vop_deleteextattr_args *ap)
 	int err;
 
 	if (fuse_isdeadfs(vp))
-		return (ENXIO);
+		return (EXTERROR(ENXIO, "This FUSE session is about to be closed"));
 
 	if (fsess_not_impl(mp, FUSE_REMOVEXATTR))
-		return EOPNOTSUPP;
+		return (EXTERROR(EOPNOTSUPP, "This daemon does not implement extattrs"));
```

Comment:
The server might well implement reading them, but not removing them.
```suggestion
		return (EXTERROR(EOPNOTSUPP, "This server does not implement removing extended attributes"));
```